ignite/config/chain/v1/validator.go

```diff
@@ -9,6 +9,8 @@ type Validator struct {
 	// Name is the name of the validator.
 	Name string `yaml:"name" doc:"Name of the validator."`
 
+	OperatorAddress string `yaml:"operatoraddress" doc:"OperatorAddress of the validator."`
```

Comment:
Shall we derive that from the normal accounts (alice/bob)?
This doesn't seem too necessary